code/modules/projectiles/projectile.dm

```diff
@@ -656,7 +656,7 @@ So if we are on the 32th absolute pixel coordinate we are on tile 1, but if we a
 		return TRUE
 	if(proj.distance_travelled <= proj.ammo.barricade_clear_distance)
 		return FALSE
-	. = coverage //Hitchance.
+	var/hit_chance = coverage //base chance for the projectile to hit the object instead of bypassing it
```

Comment:
why do you even need a new var when coverage does the trick

Reply:
Also there should be no difference between . and a "normal" var.

If there is, you really need to report that to lummox

Reply:
I bugreported it.

The var is modified before being returned. If I use coverage directly that would be permanently changing the coverage var of the object.

Reply:
Oh he actually responded. 
![image](https://user-images.githubusercontent.com/7869430/182046058-d11f5bb3-bc4b-4c51-b188-d43110852882.png)